ui/src/shared/components/AutoRefresh.js

```diff
@@ -97,6 +97,12 @@ const AutoRefresh = ComposedComponent => {
 
       const timeSeriesPromises = queries.map(query => {
         const {host, database, rp} = query
+        // the key `database` was used upstream in HostPage.js, and since as of this writing
+        // the codebase has not been fully converted to TypeScript, it's not clear where else
+        // it may be used, but this slight modification is intended to allow for the use of
+        // `database` while moving over to `db` for consistency over time
+        let {db} = query
```

Comment:
what do you think about using `const db = _.get(query, 'db', database)`

Reply:
👍